lib/github/middleware.js

```diff
@@ -46,17 +45,22 @@ module.exports = function middleware (callback) {
       repo: (context.payload.repository) ? context.repo() : undefined
     })
 
+    const gitHubInstallationId = context.payload.installation.id
+    context.log = context.log.child({ gitHubInstallationId })
```

Comment:
@tebriel In theory, this should set the `gitHubInstallationId` on all subsequent logger calls. I'm going to tweak this a little more though.